.github/workflows/astro_deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Astro deploy
+
+on:
+  push:
+    branches:
+      # Push events on main branch
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: |
+          npm run build
+          touch ./dist/.nojekyll
+
+      - name: Deploy to gh-pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this already added to the secrets part, or is already provided by GitHub? 🤔

Reply:
It's provided by GitHub :)